pkg/config/remote/client/client.go

```diff
@@ -143,6 +166,16 @@ func NewGRPCClient(ipcAddress string, cmdPort string, authTokenFetcher TokenFetc
 	return newClient(grpcClient, opts...)
 }
 
+// NewHAGRPCClient creates a new client that does not perform any TUF verification and gets failover configs via grpc
```

Comment:
For my own edification: why wouldn't we want the normal amount of payload verification that RC generally provides?

Reply:
Discussing offline, but this comment did also make me realize the func name wasn't consistent with other, so I've gone ahead and updated it to be more self descriptive.